docs/facturacion/tipos_facturas.md

```diff
@@ -52,9 +52,9 @@ La funció de les factures Anul·ladores amb Rectificadora és la d'anul·lar la
 original, de manera que el balanç sigui 0.
 
 !!! note
-A partir del 31/07/2017 aquestes factures ja no es fan servir per enviar les
```

Comment:
These changes have been done (but not merged) in [mroig PR](https://github.com/gisce/powerp-docs/pull/230)